src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1939,4 +1939,10 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="CancellingOperation" xml:space="preserve">
     <value>Cancelling operation</value>
   </data>
+  <data name="SkipDependenciesArgumentDescription" xml:space="preserve">
+    <value>Skip dependencies</value>

```

Comment:
nit: I think WindowsFeatures also honoring this? *ArgumentDescription strings should be more descriptive

Reply:
Added description for package dependencies and windows features